easy_my_coop_website_portal/views/easy_my_coop_website_portal_templates.xml

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright 2015-2016 Odoo S.A.
+     Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
remove non contributor of this module

Reply:
I have put their names because of the template `portal_my_home_menu_capital_release` that comes from their code in website_portal_v10.